### ISSUE TITLE
RCS0005 reports False Positive when #endregion follows a #pragma line plus a blank line

### ISSUE THREAD

**Issue by Bergam64** [dotnet/roslynator#1528]
**Product and Version Used**: 
VS extension **Roslynator 2022** v4.12.5

**Steps to Reproduce**:
- Enable [`RCS0005`](https://josefpihrt.github.io/docs/roslynator/analyzers/RCS0005) in your `.editorconfig` file (disabled by default).
- Open a C# file and follow example below.

**Actual Behavior**:
```csharp
namespace MyNamespace
{
    /// <summary>
    /// ...
    /// </summary>
    public class Class1
    {
        #region MyRegion

#pragma warning disable 1591

        public int MyProperty { get; set; }

#pragma warning restore 1591

        #endregion // <-- RCS0005 is wrongly reported here, although there's a blank line above.
    }
}
```

Applying the code fix adds a blank line that is not needed, which causes `RCS1036` (Remove unnecessary blank line) to be reported on that line:
```csharp
namespace MyNamespace
{
    /// <summary>
    /// ...
    /// </summary>
    public class Class1
    {
        #region MyRegion

#pragma warning disable 1591

        public int MyProperty { get; set; }

#pragma warning restore 1591


        #endregion
    }
}
```

**Expected Behavior**:
```csharp
namespace MyNamespace
{
    /// <summary>
    /// ...
    /// </summary>
    public class Class1
    {
        #region MyRegion

#pragma warning disable 1591

        public int MyProperty { get; set; }

#pragma warning restore 1591

        #endregion // <-- No RCS0005 reported here.
    }
}
```
